nexus/src/app/sagas/switch_port_settings_apply.rs

```diff
@@ -223,7 +223,7 @@ async fn spa_undo_ensure_switch_port_settings(
     let log = sagactx.user_data().log();
 
     let port_id: PortId = PortId::from_str(&params.switch_port_name)
-        .map_err(|e| external::Error::internal_error(e))?;
+        .map_err(|e| external::Error::internal_error(e.to_string().as_str()))?;
```

Comment:
is this the typify change to replace the `Result<T, String>` with a real error type?

Reply:
The `e` here seems to have type `ConversionError`, so I think that's right. Is simply stringifying it the best approach?

Reply:
Yeah I think the error is really just a Cow<'static, str>